### ISSUE TITLE
Bring back the detailed text "Start" or "Stop scanning" on the scan button

### ISSUE THREAD

**Issue by Valeryan24** [mozilla/MozStumbler#331]
Hello,

Thanks for the 0.10.0 version which corrects well the UI and behavior of 0.9.0, adding also the number of visible and scanned cells to the screen !

Just, on the scan button which is fortunately back, is it possible to re-put the same text as before ? Now in French version (I don't know on English one if it is the same) it shows only YES (Oui) or NO (Non), which can be confusing.

If Yes is written, does it mean that MozStumbler is acive and scanning, or that it will be if I click on "Yes" ? At start, when application needs some time to detect GPS, user can doubt. And the bottom blue or gray bar is not visible from car driving position.

The previous texts : "Start" or "Stop scanning" (or detection, or analysis, or activity) were perfect :)

**Comment by chpio** [comment]
Also the button is reset after every restart of the app:
1. Open the app
2. start scanning => button says: scanning
3. close the app with back-button(homebutton works)
4. reopen the app => button says: not scanning

**Comment by Valeryan24** [comment]
According to your comment, it seems that English 0.10.0 version is a bit more precise than French one (just Yes or No) for the Scan button activity, perhaps the French translation could be updated (see #321).

**Comment by pl6025** [comment]
For the French translation, it wasn't update because I've made an error in the code (forget the "\" before the " ' "). But the text of the button is not in the file strings.xml in values, so we can't localized it.

@Valeryan24 you says that you have Yes/No ? Personally I have ON/OFF. And at the launching of the app I see the OFF button, and I need to click on it to launch the scan. Very confusing.

Is it normal that we have not the same text on the button, depending of the phone ?

**Comment by Valeryan24** [comment]
@pl6025 Here is what I get on HTC Desire HD, CyanogenMod 11, Android 4.4.2, Moz Stumbler v. 0.10.0 updated from Github version :

![mozstumbler-10](https://f.cloud.github.com/assets/1119619/1983470/032918d2-8416-11e3-9591-f1a68f3ef6fb.png)

**Comment by pl6025** [comment]
@Valeryan24 Here is what I have on Sony Ericsson Xperia X8, Rom GingerDxv029 (based on CyanogenMod 6 or 7), Android 2.3.7, Moz Stumbler v 0.10.0 updated from the 0.8.3 application.

![off-on-version](https://f.cloud.github.com/assets/4056260/1983542/6ab61eb8-8417-11e3-9eed-c857fb49eebc.png)

**Comment by cpeterso** [comment]
When I added the button back, I opted to use Android's standard toggle button. The ON/OFF/OUI/NON text is Android's default text. I guess it doesn't make sense for all languages in this context.

The difference between the screenshots is that @Valeryan24 appears to be running Gingerbread and @pl6025 appears to be running Ice Cream Sandwich or Jelly Bean. Google appears to have changed the default toggle button text in these Android versions.
